src/agent/sqlite_manager/CMakeLists.txt

```diff
@@ -12,8 +12,11 @@ find_package(fmt REQUIRED)
 find_package(SQLiteCpp REQUIRED)
 
 add_library(SQLiteManager src/sqlite_manager.cpp)
+
+include(../../cmake/ConfigureTarget.cmake)
 configure_target(SQLiteManager)
-target_include_directories(SQLiteManager PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include PRIVATE ${SQLiteCpp_INCLUDE_DIRS})
+
+target_include_directories(SQLiteManager PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include ${SQLiteCpp_INCLUDE_DIRS})
```

Comment:
Is this change necessary? Remove the PRIVATE keyword?

Reply:
I believe it was needed at some point when trying to build the components separately. But since we are leaving that for further down the road, I just reverted the change.